patch/config-cig-cs6436-serail.patch

```diff
@@ -0,0 +1,65 @@
+diff -urpN a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
+--- a/debian/build/build_amd64_none_amd64/.config	2018-11-08 16:50:34.535066857 +0800
++++ b/debian/build/build_amd64_none_amd64/.config	2018-11-08 20:54:58.066163890 +0800
+@@ -3458,7 +3458,7 @@ CONFIG_DEVMEM=y
+ #
+ CONFIG_SERIAL_EARLYCON=y
+ CONFIG_SERIAL_8250=y
+-# CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
++CONFIG_SERIAL_8250_DEPRECATED_OPTIONS=y
+ CONFIG_SERIAL_8250_PNP=y
+ CONFIG_SERIAL_8250_FINTEK=y
+ CONFIG_SERIAL_8250_CONSOLE=y
+@@ -3470,12 +3470,12 @@ CONFIG_SERIAL_8250_RUNTIME_UARTS=4
+ CONFIG_SERIAL_8250_EXTENDED=y
+ CONFIG_SERIAL_8250_MANY_PORTS=y
+ CONFIG_SERIAL_8250_SHARE_IRQ=y
+-# CONFIG_SERIAL_8250_DETECT_IRQ is not set
++CONFIG_SERIAL_8250_DETECT_IRQ=y
+ CONFIG_SERIAL_8250_RSA=y
+ # CONFIG_SERIAL_8250_FSL is not set
+ CONFIG_SERIAL_8250_DW=y
+ # CONFIG_SERIAL_8250_RT288X is not set
+-# CONFIG_SERIAL_8250_LPSS is not set
++CONFIG_SERIAL_8250_LPSS=y
+ CONFIG_SERIAL_8250_MID=y
+ CONFIG_SERIAL_8250_MOXA=m
+ 
+@@ -3960,8 +3960,8 @@ CONFIG_SENSORS_DNI_DPS460=m
+ CONFIG_SENSORS_TPS53679=m
+ CONFIG_SENSORS_UCD9000=m
+ CONFIG_SENSORS_UCD9200=m
+-CONFIG_SENSORS_DPS1900=m
+ # CONFIG_SENSORS_ZL6100 is not set
++CONFIG_SENSORS_DPS1900=m
+ # CONFIG_SENSORS_SHT15 is not set
+ CONFIG_SENSORS_SHT21=m
+ # CONFIG_SENSORS_SHT3x is not set
+@@ -4169,11 +4169,11 @@ CONFIG_MFD_CORE=m
+ CONFIG_LPC_ICH=m
+ CONFIG_LPC_SCH=m
+ # CONFIG_INTEL_SOC_PMIC is not set
+-CONFIG_MFD_INTEL_LPSS=m
+-CONFIG_MFD_INTEL_LPSS_ACPI=m
+-CONFIG_MFD_INTEL_LPSS_PCI=m
++CONFIG_MFD_INTEL_LPSS=y
++CONFIG_MFD_INTEL_LPSS_ACPI=y
++CONFIG_MFD_INTEL_LPSS_PCI=y
```

Comment:
please do not change to built-in module.

Reply:
I will change it to modules and then install it during the system initialization.